include/deal.II/numerics/vector_tools.templates.h

```diff
@@ -917,7 +1050,7 @@ namespace VectorTools
       additional_data.mapping_update_flags = (update_values | update_JxW_values);
       MatrixFree<dim, Number> matrix_free;
       matrix_free.reinit (mapping, dof, constraints,
-                          QGauss<1>(fe_degree+1), additional_data);
+                          QGauss<1>(fe_degree+2), additional_data);
```

Comment:
See above (https://github.com/dealii/dealii/pull/3262/files/6141097333690dc2445211ce712a97172224b56a#diff-77183bf880358cc7866c2938080ac4b2).
So far `fe_degree+2` is used in all tests and examples.